about.json

```diff
@@ -3,7 +3,9 @@
   "about_url": null,
   "license_url": null,
   "assets": {
-    "hero_image": "assets/handshake.png"
+    "hero_image": "assets/handshake.png",
+    "dc_allert_icon_blue": "assets/icons/dc-allert-icon-blue.svg",
```

Comment:
Icons we use often are listed at https://material.io/resources/icons/?style=baseline try to use one of them or to find something similar. How to use it?

```
<span class="material-icons">
    comment
</span>
````

The above will render the comment icon in a font based way _(so you can change the colour or size easily)_

Reply:
Using an icon might not be possible due the button is a discourse component

```hbs
{{d-button action=(action "join")
    class="btn-alert"
    label="discourse_debtcollective_collectives.join"
    disabled=disabled}}
```

Do you know if I can pass something like `children` to this discourse component?

Reply:
Fixed 💯 @duranmla